src/client/Client.cc

```diff
@@ -15130,6 +15135,21 @@ void Client::handle_conf_change(const ConfigProxy& conf,
     if (cct->_conf->client_acl_type == "posix_acl")
       acl_type = POSIX_ACL;
   }
+  if (changed.count("client_oc_size")) {
```

Comment:
Please update the options in `src/common/options.cc` to have the `Option::FLAG_RUNTIME` setting.

Reply:
@batrick done.